src/palette/internal/palettecreator.cpp

```diff
@@ -779,17 +851,28 @@ PalettePtr PaletteCreator::newBreathPalette()
         SymId::fermataVeryShortAbove,
     };
 
-    for (auto i : fermatas) {
+    // Breaths
+
+    for (auto i : defaultPalette ? defaultFermatas : masterFermatas) {
         auto f = Factory::makeFermata(gpaletteScore->dummy());
         f->setSymId(i);
         sp->appendElement(f, f->userName());
     }
 
-    for (BreathType bt : Breath::breathList) {
+    for (unsigned int i = 0; i < Breath::breathList.size() - 1; i++) { //Last breath is not a default breath. Hence, - 1
```

Comment:
You should use size_t instead of unsigned int here. Also, it is better to skip elements explicitly (not by size() - 1)

Reply:
I used range-based for loop to resolve this and the following review. Hope that's okay